src/commands/convert/trivy2hdf.ts

```diff
@@ -2,6 +2,8 @@ import {Command, Flags} from '@oclif/core'
 import fs from 'fs'
 import {ASFFResults as Mapper} from '@mitre/hdf-converters'
 import {checkSuffix} from '../../utils/global'
+import _ from 'lodash'
+import path from 'path'
 
 export default class Trivy2HDF extends Command {
   static usage = 'convert trivy2hdf -i <asff-finding-json> [--securityhub <standard-1-json> ... <standard-n-json>] -o <hdf-scan-results-json>'
```

Comment:
Documentation should be changed to state that the HDFs are gonna be put into a folder and the example should be modified to state that a folder name should be provided, not a filename ending in .json.